README.md

```diff
@@ -0,0 +1,12 @@
+### Testing added
```

Comment:
We're expecting a full readme, including how to clone, install, format and test the product.